content/governance.md

```diff
@@ -145,6 +147,34 @@ A vote on a single proposal is considered successful if at least two thirds of t
 
 If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
 
+## On- / Offboarding
+
+Changes in the On- /Offboarding section can be taken by [lazy consensus](#consensus) unless challenged.  If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### Onboarding
+
+The new member is
+* added to the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR.
+* announced on the [developers mailing list][devs] by an existing team member. Ideally, the new member replies in this thread, acknowleding team membership.
+* added to the [GitHub organization][gh] as _Owner_.
+* optinally added to the community, junkyard, and related repositories.
+* added to the [team mailing list][team].
+* announced to CNCF.
+* given access to the shared password storage.
+
+### Offboarding
+
+The ex-member is
+* removed from the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR. In case of forced removal, no approval is needed.
+* removed from the [GitHub organization][gh] and related organizations and repositories.
```

Comment:
As we explicitly allow repository maintainers that are not -team members, we should also allow the transition from -team member to maintainer-only. This could come in handy if somebody wants to retire but is willing to stay a maintainer for a transition period. And it's even more relevant should we decide to sanction against too many -team members from the same company (in which case -team membership becomes a possible liability as you could "steal" voting power from your colleagues, which might make you prefer to be a maintainer without being a -team member).

Reply:
The implied statement is that we can always decide to do this anyway and have a safe default.

This question is more around "what is the default, especially when we disagree", not "what can we do if we agree".

Should I put in verbiage to make this clearer?

Reply:
Yes. I think we should distinguish between "this has to happen in any case" and "this can be done differently if we agree to".

Reply:
Added.